### PR TITLE
Added documentation text

### PR DESCRIPTION
**Pull request by EngOmbasa**
Hi @patricoferris I have added some documentation text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,5 +69,44 @@ A small example how this library is efficient in reading a `json` file _(more pa
   ]);;    
 - : (Topojson.t, [ `Msg of string ]) result = Ok <abstr>
 ```
+## An example of reading a raw JSON string
 
+To read a raw JSON string into the TopoJSON library, you need a JSON parser.In this case use Ezjsonm.
```
Comment (patricoferris) [inline README.md]: This is already explained just above.

File: README.md [hunk 2]
```diff
@@ -69,5 +69,44 @@ A small example how this library is efficient in reading a `json` file _(more pa
...
+                 "type": "Polygon"}
+            ]
+     }},
+    "transform": {"scale": [0.0005, 0.0001], "translate": [100.0, 0.0]},
+    "type": "Topology"}|};;
+val topojson_string : string =
+  "{\n    \"arcs\": [[[0.0, 0.0], [0.0, 9999.0], [2000.0, 0.0], [0.0, -9999.0], [-2000.0, 0.0]]],\n    \"objects\": {\"example \": {\n            \"type\": \"GeometryCollection\",\n            \"geometries\": [\n                {\"coordinates\": [4000.0, 5000.0],\n                 \"properties\": {\"prop0\": \"value0\"},\n     "... (* string length 595; truncated *)
+```
+
+You can then make use of the Topojson method `Topojson.of_json` that takes in the JSON string as an input and converts the it to a topojson object `(a type {!t})` or an error.
```
Comment (patricoferris) [inline README.md]:
```suggestion
You can then make use of the TopoJSON function `Topojson.of_json` that takes in the JSON string as an input and converts it to an OCaml value representing a TopoJSON object or an error. 
```

File: README.md [hunk 3]
```diff
@@ -69,5 +69,44 @@ A small example how this library is efficient in reading a `json` file _(more pa
   ]);;    
 - : (Topojson.t, [ `Msg of string ]) result = Ok <abstr>
 ```
+## An example of reading a raw JSON string
```
Comment (patricoferris) [inline README.md]:
```suggestion
#### Reading a JSON string
```

**Review by patricoferris — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (EngOmbasa) [reply README.md]: @patricoferris Noted with thanks.

**Pull request closed without merging**